README.md

```diff
@@ -45,10 +43,9 @@ With [pip](https://pip.pypa.io/en/latest/index.html) installed, run: ``pip insta
 * Structured logging initial implementation from [aws-lambda-logging](https://gitlab.com/hadrien/aws_lambda_logging)
 * Powertools idea [DAZN Powertools](https://github.com/getndazn/dazn-lambda-powertools/)
 
-
 ## Connect
 
-* **AWS Developers Slack**: `#lambda-powertools`** - **[Invite, if you don't have an account](https://join.slack.com/t/awsdevelopers/shared_invite/zt-yryddays-C9fkWrmguDv0h2EEDzCqvw)**
+* **AWS Developers Slack**: `#lambda-powertools` - **[Invite, if you don't have an account](https://join.slack.com/t/awsdevelopers/shared_invite/zt-yryddays-C9fkWrmguDv0h2EEDzCqvw)**
```

Comment:
Linting? Please revert

Reply:
Ah. I was confused by how this looked. With the "**" after the channel name.

Reply:
What is looks like now:

<img width="722" alt="Screen Shot 2022-01-11 at 11 05 57 PM" src="https://user-images.githubusercontent.com/5442469/149080045-ccb047b9-9a7e-4413-9e21-1345528d1a54.png">

Reply:
That's odd, I had fixed that before that's I got confused. Maybe it accidentally made it back in one of the previous PRs.